src/main/java/uk/gov/hmcts/reform/ccd/document/am/service/impl/DocumentManagementServiceImpl.java

```diff
@@ -199,7 +199,8 @@ public boolean patchDocumentMetadata(DocumentMetadata documentMetadata, String s
             HttpEntity<LinkedMultiValueMap<String, Object>> requestEntity = new HttpEntity<>(bodyMap, headers);
 
             restTemplate.setRequestFactory(new HttpComponentsClientHttpRequestFactory());
-            restTemplate.exchange(documentURL.concat("/documents"), HttpMethod.PATCH, requestEntity, Void.class);
+            String documentUrl = String.format("%s/documents", documentURL);
```

Comment:
I am using the formatter as junit doesn't seem to like the .concat() method. Maybe there is a way around this but this is easier.

Reply:
Yes, it looks more cleaner.